build.rs

```diff
@@ -17,13 +17,20 @@ fn gl_generate() {
                 "EGL_EXT_create_context_robustness",
                 "EGL_KHR_create_context_no_error",
                 "EGL_KHR_no_config_context",
+                "EGL_EXT_device_base",
+                "EGL_EXT_device_enumeration",
+                "EGL_EXT_device_query",
+                "EGL_EXT_device_drm",
+                "EGL_KHR_stream",
+                "EGL_KHR_stream_producer_eglsurface",
```

Comment:
I do not think we need the last two for this feature?

Reply:
Oh right, that was me. Probably an oversight when copying parts of the `EGLDevice` code from my eglstream implementation.